magnum_capi_helm/driver.py

```diff
@@ -519,6 +528,42 @@ def _get_image_details(self, context, image_identifier):
     def _get_app_cred_name(self, cluster):
         return driver_utils.get_k8s_resource_name(cluster, "cloud-credentials")
 
+    def _get_etcd_config(self, cluster):
+        # Support new-style and legacy labels for volume size and type, with
+        # new-style labels taking precedence
+        etcd_size = self._get_label_int(
+            cluster,
+            "etcd_blockdevice_size",
+            self._get_label_int(cluster, "etcd_volume_size", 0),
+        )
+        if etcd_size > 0:
+            etcd_block_device = {"size": etcd_size}
+            # The block device type can be either local or volume
+            etcd_bd_type = self._label(
+                cluster, "etcd_blockdevice_type", "volume"
+            )
+            if etcd_bd_type == "local":
```

Comment:
won't this miss if the user inputs "Local" as a value here?

Reply:
Lets do that in a follow up, if needed.